app/assets/stylesheets/common/base/user.scss

```diff
@@ -972,3 +972,23 @@
     }
   }
 }
+
+// Due to special login animations, the order here needs to be changed
```

Comment:
Oh, interesting... so then do we need to change the DOM order at all?

Reply:
It's a little confusing, but yes we do here. For the templates here, the regular fields are all hard-coded and not re-using a component. So the labels are *above* the inputs both in code & visually.

For the login and sign up section though, they are hard-coded to be under the input. It is the same with the custom user-field inputs, but since we are re-using the user-field component in the user profile page and *not* hard-coding them there, I needed to adjust the order only for that page to match the above area.